fmt/printf.ts

```diff
@@ -611,7 +621,10 @@ class Printf {
           prefix += "0x";
           break;
         default:
-          throw new Error("cannot handle base: " + radix);
+          throw new Error(
+            "Format number only supports radix 2, 8 and 16: cannot handle radix " +
```

Comment:
I'd suggest the below

```suggestion
            `Cannot handle the radix ${radix}. Only 2, 8, 16 are supported`
```

because of the rule `Message should state the action that lead to the error`

Reply:
I've merged a slightly different version of this that reads: `Cannot handle the radix ${radix}: only 2, 8, 16 are supported`